README.md

```diff
@@ -207,6 +207,17 @@ this can be disabled on an individual basis by adding
 `<ignoreSubpackages>false</ignoreSubpackages>` to the `<ignoreSourcePackage>`
 or `<ignoreDestinationPackage>` element.
 
+## Include dependencies with different scopes
+
+NOTE: this feature is not available in the released version, only in the latest snapshot.
+
+In general, it only makes sense to run missinglink for an executable artifact as opposed to a
+library. The reason is that the exact set of dependencies that are included can change every time
```

Comment:
I am not sure if I follow this totally or agree with the idea using missinglink on a library doesn't make sense. 

A library might depend on two other libraries which have a shared dependency (let's say Guava) which had a breaking change between the version A was compiled against and the version B was compiled against. The library could depend on something in A or B which would break at runtime depending on which version of Guava was actually resolved.

Of course the "final" project using this library might introduce some new variation in the dependency tree that changes which version of Guava is resolved, perhaps in a way that causes the library to not have issues, or causes new issues. So I would say if a "final" project uses this library then it isn't enough, from the perspective of the final project, for the library to be checked with missinglink - ideally both should be scanning for issues like this.